File: Examples/CSharp/Articles/FilteringObjectsAtLoadTime/FilteringObjects.cs
using System;

namespace Aspose.Cells.Examples.CSharp.Articles.FilteringObjectsAtLoadTime
{
    class FilteringObjects
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Filter charts from the workbook
            LoadOptions lOptions = new LoadOptions();
            lOptions.LoadFilter = new LoadFilter(LoadDataFilterOptions.All & ~LoadDataFilterOptions.Chart);

            // Load the workbook with above filter
            Workbook workbook = new Workbook(sourceDir + "sampleFilteringObjects.xlsx", lOptions);

            // Save worksheet to a single PDF page
            PdfSaveOptions pOptions = new PdfSaveOptions();
            pOptions.OnePagePerSheet = true;

            // Save the workbook in pdf format
            workbook.Save(outputDir + "outputFilteringObjects.pdf", pOptions);

            Console.WriteLine("FilteringObjects executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/FilteringObjectsAtLoadTime/FilteringObjects.vb
Namespace Articles.FilteringObjectsAtLoadTime
    Friend Class FilteringObjects
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Create directory if it is not already present.
            Dim IsExists As Boolean = System.IO.Directory.Exists(dataDir)
            If (Not IsExists) Then
                System.IO.Directory.CreateDirectory(dataDir)
            End If

            ' Filter charts from the workbook
            Dim lOptions As New LoadOptions()
            lOptions.LoadFilter = New LoadFilter(LoadDataFilterOptions.All And (Not LoadDataFilterOptions.Chart))

            ' Load the workbook with above filter
            Dim workbook As New Workbook(dataDir & "sampleFilterCharts.xlsx", lOptions)

            ' Save worksheet to a single PDF page
            Dim pOptions As New PdfSaveOptions()
            pOptions.OnePagePerSheet = True

            ' Save the workbook in pdf format
            workbook.Save(dataDir & "sampleFilterCharts.pdf", pOptions)
            ' ExEnd:1

        End Sub
    End Class
End Namespace